app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -53,7 +60,7 @@
 
     <% if dismiss_text %>
       <p class="govuk-body">
-        <%= tag.a class: "govuk-link", href: dismiss_href, data: dismiss_data_attributes do %>
+        <%= tag.a class: "govuk-link gem-c-intervention__dismiss-link", href: dismiss_href do %>
```

Comment:
Since the `gem-c-intervention__dismiss-link` class is used entirely as a JS hook and not for styling, could it be renamed to indicate this? Something like `js-dismiss-link` would be fine.

Reply:
thanks - done